contracts/mirrorens.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.4.0;
```

Comment:
the mirrorENS contract should be a generic one 
where you would register the namehash pair for the serviceId
then the testimonyFor call gives you the serviceId which you use to look up the ens-s to be compared